languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -1073,6 +1073,76 @@ USA
             <example correction="we're">Trying to figure-out if <marker>were're</marker> going to Swansboro</example>
             <example correction="We're"><marker>Were're</marker> making a big deal out of nothing.</example>
         </rule>
+        <rule id="HAS_IT_NNS__IT_ITS" name="has it advantages → has its advantages" default="temp_off">
+            <antipattern><!-- There we have it -->
+                <token>there</token>
+                <token postag="PRP">
+                    <exception case_sensitive="yes" regexp="yes">[Ii]t</exception>
+                </token>
+                <token>have</token><!-- TODO: inflected? -->
+                <token>it</token>
+            </antipattern>
+            <!-- Copied antipattern from MAKE_PRP_VBZ -->
+            <antipattern><!-- For every room I have it bundles all lights. -->
+                <token>for</token>
+                <token skip="-1" postag="DT" regexp="yes">each|every|all|both|most|some|either|neither</token>
+                <token postag_regexp="yes" postag="VBN?" />
+                <!-- Ideally insert a comma -->
+                <token postag="PRP" />
+            </antipattern>
+
+            <pattern>
+                <token inflected="yes">have
+                    <exception scope="previous" postag="SENT_START" />
+                    <exception scope="previous">that</exception><!-- TODO: overzealous -->
+                    <exception scope="previous">nor</exception>
+                    <exception scope="previous" regexp="yes">(?:rumou?r|legend|tradition)s?|buzz|word|wisdom|folklore</exception>
+                </token>
+                <marker>
+                    <token case_sensitive="yes">it</token>
+                </marker>
+                <!-- moments origins quirks charms benefits strengths positives advantanges privileges ups pros virtues
+                flaws limitations shortcomings problems negatives cons downs downsides drawbacks disadvantages weaknesses perils pitfalls
+                peculiarities nuances ways motives setbacks proponents similarities resemblances -->
+                <!-- Disambiguation rule PRP_VBZ removes NNS -->
+                <token postag_regexp="yes" postag="NNS|VBZ">
+                    <exception regexp="yes" inflected="yes">be|have|seem|make</exception>
+                    <exception>backwards</exception><!-- RB -->
+                    <exception>his</exception><!-- PRP$ -->
+                    <exception>things</exception>
+                    <exception scope="next" spacebefore="no" regexp="yes">&apostrophe;</exception><!-- UPS'd -->
+                </token>
+            </pattern>
+            <message>Did you mean <suggestion>its</suggestion>?</message>
```

Comment:
I would like to see a short explanatory sentence here, along the lines of "The word 'it' doesn't seem to fit here, did you mean..."